### DISCUSSION TITLE
processor over loading and delay while playing on macbook m3 pro. ( note : i know what buffer size is / I am using a gigabit wired network)

### DISCUSSION THREAD

**Discussion by sameeraonlinex** [apohl79/audiogridder#1405] [Help]
I am using AudioGridder with a plugin inside a MacBook M3 Pro (18 GB RAM) as the host, and the server is a Windows Core i5 8th Gen PC. I am trying to play a very simple piano plugin, but it keeps showing that the processor is overloading on Cubase's CPU meter. I tried changing the buffer sizes in Cubase, as well as in the plugin itself, and I also tried adding manual buffer delays from the AudioGridder app. However, it’s still overloading. Even at a 1024 buffer size, I hear crackling sounds. What are the solutions? I love this project!

**Comment by bluetardis** [comment]
Start by making sure that they are really using the gigabit wired network (eg wifi is off for both computers).

**Reply by sameeraonlinex** [reply]
yes definitely it is a gigabit network. I used these switches and cat6 cables for DANTE networks as well. and it is not on the wifi. Finally i bought Vienna Ensemble Pro 7, that works fine on no Buffer mode.
